config/after_symlink.yml

```
@@ -12,3 +12,6 @@
 
 - include_tasks: "../../lephare.ansible-deploy/config/steps/sentry_notify.yml"
   when: lephare_sentry_notify
+
+- include_tasks: "../../lephare.ansible-deploy/config/steps/notify_slack_deploy.yml"
+  when: lephare_notify_slack_deploy
```

```suggestion
  when: lephare_slack_notify
```

deploy n'est pas nécessaire puisque tout le dépôt ne sert qu'à déployer.

---

c'est fait @erichard